drivers/dai/intel/ssp/ssp.c

```diff
@@ -1538,7 +1538,11 @@ static int dai_ssp_set_config_blob(struct dai_intel_ssp *dp, const void *spec_co
 	ssrsa = blob->i2s_driver_config.i2s_config.ssrsa;
 
 	sys_write32(ssc0, dai_base(dp) + SSCR0);
+	sys_write32(blob->i2s_driver_config.i2s_config.ssc2 & (~(BIT(20))),
+			dai_base(dp) + SSCR2); /* hardware specific flow */
 	sys_write32(blob->i2s_driver_config.i2s_config.ssc1, dai_base(dp) + SSCR1);
+	sys_write32(blob->i2s_driver_config.i2s_config.ssc2 | (BIT(20)),
+			dai_base(dp) + SSCR2); /* hardware specific flow */
```

Comment:
Wouldn't hurt to put that BIT(20) into the header somewhere and give it a symbolic name.  From context it looks like some kind of dis/enable bit to protect the write to SSCR1?